packages/gcf-utils/src/bin/genkey.ts

```diff
@@ -25,127 +25,133 @@ import { Base64 } from 'js-base64';
 const id: number = argv.id as number;
 
-
 if (!project) {
-    console.error('Project name is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('Project name is required');
+  yargs.showHelp();
+  process.exit(1);
 }
 if (!botname) {
-    console.error('Name of the bot is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('Name of the bot is required');
+  yargs.showHelp();
+  process.exit(1);
 }
 if (!webhookSecret) {
-    console.error('Webhook secret is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('Webhook secret is required');
+  yargs.showHelp();
+  process.exit(1);
 }
 if (!id) {
-    console.error('GitHub Application ID is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('GitHub Application ID is required');
+  yargs.showHelp();
+  process.exit(1);
 }
 if (!bucketName) {
-    console.error('Bucket Name is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('Bucket Name is required');
+  yargs.showHelp();
+  process.exit(1);
 }
 
-let keyContent: string = '';
+let keyContent = '';
 try {
-    keyContent = fs.readFileSync(keyfile, 'utf8');
+  keyContent = fs.readFileSync(keyfile, 'utf8');
 } catch (e) {
-    console.log(`Error reading file: ${keyfile}`);
-    process.exit(1);
+  console.log(`Error reading file: ${keyfile}`);
+  process.exit(1);
 }
 
-let blob: Options = {
-    cert: keyContent,
-    id: id,
-    secret: webhookSecret
-}
+const blob: Options = {
+  cert: keyContent,
+  id,
+  secret: webhookSecret,
+};
 
 async function run() {
-    let encblob: Buffer = Buffer.from('');
-    try {
-        const opts = project ? { projectId: project } as KMS.v1.KeyManagementServiceClient.ConfigurationObject : undefined;
-
-        const kmsclient = new KMS.KeyManagementServiceClient(opts);
-
-        const name = kmsclient.cryptoKeyPath(
-            project,
-            location,
-            keyring,
-            botname
-        );
-
-        let plaintext = Base64.encode(JSON.stringify(blob));
-        const [kmsresult] = await kmsclient.encrypt({ name, plaintext: plaintext });
-        encblob = kmsresult.ciphertext;
-
-    } catch (e) {
-        console.error('Got an error encrypting the blob');
-        console.error(e);
-        process.exit(1);
-    }
+  let encblob: Buffer = Buffer.from('');
+  try {
+    const opts = project
+      ? ({
+          projectId: project,
+        } as KMS.v1.KeyManagementServiceClient.ConfigurationObject)
+      : undefined;
+
+    const kmsclient = new KMS.KeyManagementServiceClient(opts);
+
+    const name = kmsclient.cryptoKeyPath(project, location, keyring, botname);
+
+    const plaintext = Base64.encode(JSON.stringify(blob));
+    const [kmsresult] = await kmsclient.encrypt({ name, plaintext });
+    encblob = kmsresult.ciphertext;
+  } catch (e) {
+    console.error('Got an error encrypting the blob');
+    console.error(e);
```

Comment:
Instead of this, can we just `throw e`?  @bcoe

Reply:
@orthros in this case, I tend to agree, you could probably just drop the `try`, `catch`, and let the exception bubble to the terminal.

Where I will sometimes capture errors, is for user facing CLI applications, where I potentially want to obfuscate the internal implementation (do they even need to know it's Node.js). In this case, I'd just let it throw.

Reply:
Done 👍